devel/pika-algorithms/Portfile

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           boost 1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+PortGroup           mpi 1.0
+
+# Keep in sync with Pika:
+boost.version       1.81
+
+github.setup        pika-org pika-algorithms 0.1.4
+revision            0
+categories          devel parallel
+platforms           {darwin any}
+license             Boost-1
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         ${name} is a C++ library containing parallel algorithms built on top of pika
+long_description    {*}${description}
+checksums           rmd160  1063dd5aac28ff6d3e803205a80324abe1901b8a \
+                    sha256  67ea5e8545b234f82dcc75612a774f2e3df8425a283f2034c2d1e2e5ac74f945 \
+                    size    832806
+github.tarball_from archive
+supported_archs     noarch
```

Comment:
why would this be noarch - there are clearly things being compiled...

Reply:
It installs headers and CMake configuration files, no binaries or libs.
I should perhaps check if anything arch-specific is getting written down into CMake files though in a text form.

Reply:
why is there all the mpi stuff and compiler selection going in if nothing is being build. That seems all very counter intuitive.

Reply:
I made the settings to match those of Pika itself (which does use MPICH etc), and configure of this port checks for all that stuff.
It also does compile things if tests are built.